Chronos/Views/AmountsChart.swift

```diff
@@ -125,40 +133,74 @@ struct AmountsChart: View {
                     .fontWeight(.bold)
         }
     }
+    /// Workaround to occasional hashing error
```

Comment:
Here is a good example for why you should be making a new commit for each change. If someone wants to refactor this two years from now, they'll see this message and they'll wonder if what they think is a good way to simplify this file will actually reintroduce this error you tried to work around. The way the will avoid that is by looking at the commit that introduced this message to see what the error was that it worked around. But since you introduced the error, noticed you had made an error, and then fixed it, all without ever stopping to commit your changes, what the error was is now lost to time and the comment becomes way less useful.